test/E2ETests/Implementation/Validator.cs

```diff
@@ -29,6 +36,64 @@ public partial class Validator
             _deploymentResult = deploymentResult;
         }
 
+        private Task<HttpResponseMessage> DoGetAsync(string uri)
+            => DoGetAsync(new Uri(uri, UriKind.RelativeOrAbsolute));
+
+        private async Task<HttpResponseMessage> DoGetAsync(Uri uri)
+        {
+            var response = await _httpClient.GetAsync(uri);
+            SaveHttpOnlyCookies(response);
+            return response;
+
+        }
+
+        private  Task<HttpResponseMessage> DoPostAsync(string uri, HttpContent content)
+            => DoPostAsync(new Uri(uri, UriKind.RelativeOrAbsolute), content);
+
+        private async Task<HttpResponseMessage> DoPostAsync(Uri uri, HttpContent content)
+        {
+            var response = await _httpClient.PostAsync(uri, content);
+            SaveHttpOnlyCookies(response);
+            return response;
+        }
+
+        private void SaveHttpOnlyCookies(HttpResponseMessage response)
```

Comment:
@kichalla still don't understand fully why this solves the issue...probably worth investigating more.